api/src/test/java/uk/gov/hmcts/payment/api/componenttests/SwaggerPublisher.java

```diff
@@ -63,7 +63,7 @@ public void generateDocs() throws Exception {
 
     private void generateDocsForGroup(String groupName) throws Exception {
         byte[] specs = mvc.perform(
-            get("/v2/api-docs?group=" + groupName)
+            get("/v3/api-docs?group=" + groupName)
```

Comment:
Note that the Swagger is not actually even published automatically as part of an Action - should really be fixed at some point. The current config in here doesn't quite work as expected (it generates multiple files as intended, but they all have the same content with ALL endpoints, rather than being grouped) but not related to these changes and since not published, this can probably be resolved separately?